app/Services/SentinelTrafficClient.php

```
@@ -0,0 +1,180 @@
+        return instant_remote_process(
+            ["docker exec coolify-sentinel sh -c 'curl -H \"Authorization: Bearer {$token}\" {$url}'"],
+            $this->server,
+            false
+        );
+    }
+
+    private function guard(string $response): string
+    {
+        if (str($response)->contains('"error"')) {
+            $error = data_get(json_decode($response, true), 'error', 'Traffic analytics request failed.');
+            throw new \Exception($error);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '175', column '23'). (undefined)

(MissingImport)